### PR TITLE
Set up server

### PR DESCRIPTION
**Pull request by MohamedOmarii**
related to #8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "nazareth-events",
+  "version": "1.0.0",
+  "description": "A web app for viewing events in Nazareth",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon ./start.js"
```
Comment (mattlub) [inline package.json]: should have `node ./start.js` as start script (I think) just for when it is on Heroku, it runs the `start` script.
You can add a separate `dev` script

File: start.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const server = require('./server.js');
+const http = require('http');
+
+http.createServer(server).listen(server.get ('port'), function (){
```
Comment (mattlub) [inline start.js]: you don't need to do this, with express you can just do `server.listen(...`

see: https://expressjs.com/en/starter/hello-world.html

**Comment by mattlub** [comment]
also remember to always add a description to the PR, and reference the issue.

File: package.json (hunk 1 above)
Comment (MohamedOmarii) [inline package.json]: :+1:

File: start.js (hunk 2 above)
Comment (MohamedOmarii) [inline start.js]: you means the http.createServer ?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "nazareth-events",
+  "version": "1.0.0",
+  "description": "A web app for viewing events in Nazareth",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start":"node ./start.js",
+    "dev": "./node_modules/.bin/nodemon start.js",
+    "lint": "./node_modules/.bin/eslint '**/**/*.js'"
```
Comment (mattlub) [inline package.json]: eslint is not installed in the dev dependencies.

Comment (MohamedOmarii) [inline package.json]: I'll install it

**Pull request merged**